src/lib/datepicker/month-view.ts

```diff
@@ -98,9 +98,6 @@ export class MatMonthView<D> extends CalendarView<D> implements AfterContentInit
   /** A function used to filter which dates are selectable. */
   @Input() dateFilter: (date: D) => boolean;
 
-  /** Emits when a new date is selected. */
-  @Output() readonly selectedChange: EventEmitter<D | null> = new EventEmitter<D | null>();
```

Comment:
why are these being deleted?

Reply:
I deleted the `selectedChange` in the views because since the views are extending `CalendarView` and since `CalendarView` has the same property `selectedChange`, I thought we could avoid code duplications here.

Reply:
Ah, that should be fine but you probably need to add it to the `outputs` section of the component decorator: https://angular.io/api/core/Directive#outputs